nvflare/apis/impl/job_def_manager.py

```diff
@@ -134,10 +134,13 @@ def _validate_uploaded_content(self, uploaded_content) -> bool:
         """
         pass
 
-    def get_job(self, jid: str, fl_ctx: FLContext) -> Job:
+    def get_job(self, jid: str, fl_ctx: FLContext) -> Optional[Job]:
         store = self._get_job_store(fl_ctx)
         job_meta = store.get_meta(self.job_uri(jid))
-        return job_from_meta(job_meta)
```

Comment:
Instead of changing the storage code, should we just add a try catch block here?